lib/sles4sap/sap_deployment_automation_framework/deployment.pm

```diff
@@ -1226,4 +1228,80 @@ sub get_fencing_mechanism {
     return ($supported_fencing_values{$fencing_type});
 }
 
+=head3 sdaf_upload_logs
+
+    sdaf_upload_logs(hostname => $hostname)
+
+    Collect and upload SDAF logs present and generated locally on SUT.
+
+=over
+
+=item B<hostname> - SUT hostname
+
+=back
+=cut
+
+sub sdaf_upload_logs {
+    my (%args) = @_;
+    my $hostname = $args{hostname};
+    my $bootstrap_log = '/var/log/ha-cluster-bootstrap.log';
```

Comment:
Only used once. Maybe a variable is not needed and the path can be directly used in the crm report command at line 1254

Reply:
Yes. I will use it directly.